de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/session/internal/UserInformationHandler.java

```diff
@@ -282,7 +282,7 @@ private void handleUserFinishedProjectNegotiationPacket(Packet packet) {
          * thread context and thus blocking the dispatching for network packets
          * for an unknown time.
          */
-        session.userFinishedProjectNegotiation(fromUser);
+        session.userFinishedReferencePointNegotiation(fromUser);
     }
 
     private void handleUserListUpdate(Packet packet) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Move this method into the anonymous class declared at line 54. ([squid:S3398](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS3398))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=273&issues=AWcmJq0WkEeD8QRkTSuW&open=AWcmJq0WkEeD8QRkTSuW)

[ISSUE_KEY]: AWcmJq0WkEeD8QRkTSuW